src/components/cards/ShsHacksCard.vue

```diff
@@ -1,21 +1,23 @@
 <template>
-  <card>
-    <a class="main-link" href="https://docs.google.com/forms/d/e/1FAIpQLSeKScB0_C3Esobhakie0zaI9jqeejckgXy4xoCjE-DywjgJbw/viewform?usp=sf_link" target="_blank">
-      <img class="logo" src="@/assets/occasions/shsHacks.png">
-      <div class="register">
-        Stevenson's free-to-attend hackathon is happening <span class="bold">Saturday March 2nd!</span> <br><span class="color">Register Now!</span>&nbsp;
-        <font-awesome-icon :icon="faArrowUpRightFromSquare" class="link-icon" />
-      </div>
-    </a>
-  </card>
+  <timed-card startTime="February 20, 2024" endTime="March 2, 2024">
```

Comment:
Just since this card is so large, it will likely be intrusive to have it appear for the next week and a half. Can this be adjusted to this weekend/next week? Feel free to pick either or - up to you